addons/aircraft/XEH_postInitClient.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+
+if (!hasInterface) exitWith {};
+
+private _cfgAction = configFile >> "CfgActions" >> "Eject";
+GVAR(ejectActionParams) = [
+    [
+        "", // will be set with setUserActionText
+        {
+            params ["_vehicle", "_unit"];
+            private _preserveEngineOn = (_unit == driver _vehicle) && {isEngineOn _vehicle};
+            moveOut _unit;
+            if (_preserveEngineOn) then {
+                // vehicle is local to last driver, no need to care
+                _vehicle engineOn true;
+            };
+        },
+        nil,
+        getNumber (_cfgAction >> "priority"),
+        false,
+        true,
+        getText (_cfgAction >> "shortcut"),
+        QUOTE([ARR_2(_this,_target)] call DFUNC(canShowEject))
```

Comment:
Just a tip, you could've written this as:
```sqf
'[_this, _target] call DFUNC(canShowEject)'
```
This is fine too though.

Reply:
that's amazing